src/epub/text/status-quo.xhtml

```diff
@@ -98,7 +98,7 @@
 			<p>Steve came to his feet and shook hands. “Thought that you were going to be down in Florida bass fishing this month. You like your work so well you can’t stay away, or is it a matter of trying to impress your chief?”</p>
 			<p>Larry growled, “Fine thing. Secret Service bogs down and they’ve got to call me in to clean up the mess.”</p>
 			<p>Steve motioned him to a chair and immediately went serious. “Do you know anything about pushing queer, Woolford?”</p>
-			<p>“That means passing counterfeit money, doesn’t it? All I know is what’s in the <abbr epub:type="z3998:acronym">TriD</abbr> crime shows.”</p>
+			<p>“That means passing counterfeit money, doesn’t it? All I know is what’s in the <Tri-D crime shows.”</p>
```

Comment:
Broken HTML here and in all other instances

Reply:
Sorry. fixed